src/particle.h

```diff
@@ -26,6 +26,8 @@ class Particle : protected Pointers {
   int sorted;               // 1 if particles are sorted by grid cell
 
   enum{MAXVIBMODE=4};       // increase value if species need more vib modes
+  enum{MAXELECSTATE=12};      // increase value if species need more electronic states
+  enum{MAXSPECIES=16};      // increase value if more species are needed for species-specific electronic relaxation rates
```

Comment:
`MAXSPECIES` name is confusing to use inside the `Species` struct, since the `species` vector will be `nspecies` long vector of the `Species` struct, so that seems like `nspecies` elements inside `nspecies` vector.

Reply:
Yes, having a vector of nspecies inside each species struct is intended. Essentially each state and each collision partner species can have different relaxation numbers/behavior, hence the array. However, I rarely expect that array to be dense and unique (i.e. there will be many columns and/or rows which are filled with the same value). With your next comment we can almost certainly reorganize all this data to reduce the memory overhead.